public/locales/en.json

```diff
@@ -219,18 +219,44 @@
             "uploadImage": "Upload Image",
             "table": "Table",
             "line": "Horizontal line",
-            "comment": "Comment"
+            "comment": "Comment",
+            "preferences": "Editor settings",
+            "emoji": "Open emoji picker"
         },
         "menu": {
             "menu": "Menu",
+            "import": "Import",
+            "export": "Export",
             "new": "New",
-            "publish": "Publish",
+            "shareLink": "Share link",
             "extra": "Extra",
             "revision": "Revision",
             "slideMode": "Slide Mode",
             "download": "Download",
             "help": "Help",
-            "deleteNote": "Delete note"
+            "deleteNote": "Delete note",
+            "permissions": "Permissions",
+            "documentInfo": "Document info",
+            "pinNoteToHistory": "Pin note to history",
+            "pinnedToHistory": "Pinned to history"
+        },
+        "preferences": {
+            "title": "Preferences",
+            "theme": "Theme",
+            "keymap": "Keymap",
+            "tabCharacter": "Tab character",
+            "tabSize": "Tab size",
+            "spellChecker": "Spell checking"
+        },
```

Comment:
shouldn't that be under "modal" ?

Reply:
The translation keys were restructured. "preferences" is now under "modal".